dev-docs/bidders/connatix.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: bidder
+title: Connatix
+description: Connatix Bidder Adapter
+biddercode: connatix
+gdpr_supported: true

```

Comment:
Please change this field attribute to `tcfeu_supported` - we've changed since this PR was opened. Thanks.

Reply:
Done, thanks!